Meziantou.FluentAssertionsAnalyzers/NunitAssertAnalyzerCodeFixProvider.cs

```diff
@@ -543,6 +546,14 @@ private static async Task<Document> Rewrite(Document document, SyntaxNode nodeTo
                         }
                     }
 
+                    bool IsCollection(ArgumentSyntax argumentSyntax)
+                    {
+                        var argumentTypeSymbol = semanticModel.GetOperation(argumentSyntax.Expression, cancellationToken)?.Type;

```

Comment:
I think you can use `semanticModel.GetTypeInfo(argumentSyntax.Expression, cancellationToken)` instead of getting the `IOperation`. It should be a little faster.